### ISSUE TITLE
MaterialCardView rounded corners 

### ISSUE THREAD

**Issue by uktshukg** [material-components/material-components-android#968]
**Description:** Corners blurry in android 7.1.1, screenshot link : [https://imgur.com/dM26zyM](url)

**Source code:** 
```
 <com.google.android.material.card.MaterialCardView
        android:id="@+id/due_action"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:clipToPadding="true"
        app:cardElevation="0dp"
        app:strokeColor="@color/colorPrimary"
        app:cardCornerRadius="24dp"
        app:strokeWidth="1dp">

        <TextView
            android:id="@+id/due_action_text"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginLeft="13dp"
            android:layout_marginTop="7dp"
            android:layout_marginRight="13dp"
            android:layout_marginBottom="7dp"
            android:gravity="center"
            android:text=";lcsdvdlv"
            android:textAllCaps="false"
             />

    </com.google.android.material.card.MaterialCardView>
```

**Android API version:**  Api 25

**Material Library version:** implementation 'com.google.android.material:material:1.0.0'

**Device:** Moto E4 plus

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by uktshukg** [comment]
[https://imgur.com/dM26zyM](url)

**Comment by ikim24** [comment]
This appears to be fixed in 1.1 and there should be a stable version coming soon.

**Comment by uktshukg** [comment]
@ikim24  1.1.0-rc01 also has the same issue
